fs/compress.py

```diff
@@ -152,26 +160,21 @@ def write_tar(src_fs,
             tar_info = tarfile.TarInfo(tar_name)
 
             if info.has_namespace('stat'):
-                mtime = info.get('stat', 'st_mtime', None)\
-                                    or time.time()
+                mtime = info.get('stat', 'st_mtime', current_time)
```

Comment:
This is so that if the time has to be assumed, it is consistent. Other wise times will be different by some milliseconds. Quite what the impact would be, I'm not sure.